contribution.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+---
+
+## Open source
+
+The Media-Cloud AI source code is hosted on the [https://gitlab.com/media-cloud-ai](https://gitlab.com/media-cloud-ai) Gitlab group.
```

Comment:
What about giving some precisions on the structure of the group? Like mentionning what are each subgroup supposed to contain (sounds quite obvious for workers and backend but might be useful for startup/librairies/command line tools)...